packages/dd-trace/src/encode/agentless-ci-visibility.js

```diff
@@ -156,29 +155,27 @@ class AgentlessCiVisibilityEncoder extends AgentEncoder {
     const payload = {
       version: ENCODING_VERSION,
       metadata: {
-        'language': 'javascript',
-        'library_version': tracerVersion,
-        'runtime.name': 'node',
-        'runtime.version': process.version
+        '*': {
```

Comment:
What does this do? Metadata should probably always be strings or numbers to match span meta.

Reply:
'*' means the metadata here will be attach to *all* spans. We intend to make different dictionaries for different types of spans, for example, `'test'` metadata dictionary to add to all test spans

Reply:
So basically there will be no actual tags at the root, only dictionaries?

Reply:
correct